Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt

```diff
@@ -72,6 +74,7 @@ fun PlayerScreen(
 @Composable
 private fun PlayerScreen(
     playerUiState: PlayerUiState?,
```

Comment:
PlayerUiState can be non nullable.

We might want to change the model to reflect no loaded item, rather then "" being no item.

Reply:
thanks, removed the ?